recipe/meta.yaml

```diff
@@ -1,20 +1,20 @@
-{% set version = "3.4.1" %}
+{% set version = "3.4.2" %}
 
 package:
   name: conda-smithy
   version: {{ version }}
 
 source:
   url: https://github.com/conda-forge/conda-smithy/releases/download/v{{ version }}/conda-smithy-{{ version }}.tar.gz
-  sha256: 073038fd08a0a29887f20445b13d6a9ed84dfeacf1324a2aacbac9d490905a00
+  sha256: 994d8a3e841879bda8637ffb170cbe8132f28623575817f1c8d66d54ef7a3eab
   patches:
     - 1102.diff
```

Comment:
Do we still want to have this? https://github.com/conda-forge/conda-smithy/pull/1102 wasn't resolved.

Reply:
Yes, we do.  We can always issue a new release

Reply:
I'm sorry, I don't understand what you are saying here.

Reply:
Oh sorry, I misunderstood.